src/tests/programs/clinical-submission.ts

```diff
@@ -1,49 +1,46 @@
-const assert = require('assert');
-
-const {
-  afterEach,
-  startAsUser,
-  updateStatus,
-  buildUrl,
-  loginAsUser,
-  TEST_USERS,
-} = require('../../helpers');
-
-const {
+import { BaseTest, Done } from '../../types';
+
+import { startAsUser, buildUrl, TEST_USERS, submitResults } from '../../helpers';
+
+import {
   generateProgram,
   createProgram,
   registerSamples,
   submitClinicalData,
-} = require('../../utils/programUtils');
-const { multiSelectClick, selectClick, multiCheckboxClick } = require('../../utils/formUtils');
+} from '../../utils/programUtils';
+import { NightwatchBrowser } from 'nightwatch';
 
 const program = generateProgram();
 
-module.exports = {
+const ClinicalSubmissionTest: BaseTest = {
+  '@disabled': true,
+  developer: 'Ciaran Schutte',
   tags: ['programs', 'clinical-submission'],
   desiredCapabilities: {
     name: 'Registration',
   },
-  before: function(browser, done) {
+
+  before: async (browser: NightwatchBrowser, done) => {
     return createProgram({ jwt: TEST_USERS.DCC_ADMIN.token, program }).then(() => {
       done();
     });
   },
-  'Register - Empty State': browser => {
+
+  'Register - Empty State': (browser: NightwatchBrowser) => {
     startAsUser(browser)(TEST_USERS.DCC_ADMIN).url(
       buildUrl(`/submission/program/${program.shortName}/sample-registration`),
     );
     browser.pause(2500);
     browser.assert.visible('#button-register-file-select');
     browser.expect.element('#button-register-samples-commit').to.have.attribute('disabled');
   },
-  'Register - Upload Samples and Clear': browser => {
+
+  'Register - Upload Samples and Clear': (browser: NightwatchBrowser) => {
     startAsUser(browser)(TEST_USERS.DCC_ADMIN)
-      .perform(async function(done) {
+      .perform(async (done: Done) => {
         await registerSamples({
           jwt: TEST_USERS.DCC_ADMIN.token,
           shortName: program.shortName,
-          count: 5,
```

Comment:
just wanna confirm this change is intentional eh? just because this is change in value space

Reply:
yup, count isn't used by RegisterSamples